crates/types/src/services/p2p.rs

```diff
@@ -52,10 +52,10 @@ pub enum GossipsubMessageAcceptance {
     /// Report whether the gossiped message is valid and safe to rebroadcast
     Accept,
     /// Ignore the received message and prevent further gossiping
-    Reject,
+    Ignore,
     /// Punish the gossip sender for providing invalid
     /// (or malicious) data and prevent further gossiping
```

Comment:
I think the comment should mention the message. Also, we can't guarantee that they will be punished by
choosing this type, but we can "indicate" or mark them so our peer reputation can.
```suggestion
    /// Reject received message and indicate that the gossip sender has provided 
    /// invalid (or malicious) data
```

Reply:
Sounds good to me :+1: